### PR TITLE
fix the expression about creating sql user when enabling sem

### PR DESCRIPTION
**Pull request by baurine**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Refine the expression about how to create a SQL user when enabling SEM.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): #7187 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by crazycs520 — APPROVED** [review]

File: dashboard/dashboard-user.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ TiDB Dashboard 与 TiDB 使用相同的用户权限体系和登录验证方式
     GRANT SYSTEM_VARIABLES_ADMIN ON *.* TO 'dashboardAdmin'@'%';
     ```
 
-- 当所连接的 TiDB 服务器启用了[安全增强模式 (SEM)](/system-variables.md#tidb_enable_enhanced_security) 时，你可以通过执行以下示例 SQL 语句创建一个允许登录 TiDB Dashboard 的 SQL 用户 `dashboardAdmin`：
+- 当所连接的 TiDB 服务器启用了[安全增强模式 (SEM)](/system-variables.md#tidb_enable_enhanced_security) 时，你可以先关闭 SEM，通过执行以下示例 SQL 语句创建一个允许登录 TiDB Dashboard 的 SQL 用户 `dashboardAdmin` 后再重新开启 SEM：
```
Comment (breeswish) [inline dashboard/dashboard-user.md:57]: Why do we need to do this? I cannot understand the reason.

Comment (baurine) [reply dashboard/dashboard-user.md:57]: I just found after enabling the SEM, the root user can't grant `dashboardAdmin` `RESTRICTED_xxx` privileges, and it reports error:

```mysql
mysql> GRANT RESTRICTED_STATUS_ADMIN ON *.* TO 'dashboardAdmin'@'%';
ERROR 1227 (42000): Access denied; you need (at least one of) the GRANT OPTION privilege(s) for this operation
```

And the root user self has no `RESTRICTED_xxx` privileges.

Then I notice in the permission isolation docs, it says:

> Execute following SQL should disable SEM first, after execute finish, enable SEM again.

Comment (breeswish) [reply dashboard/dashboard-user.md:57]: I guess it is caused by the `root` user does not have sufficient privilege to grant privileges in the SEM mode.

**Review by breeswish — APPROVED** [review]

File: dashboard/dashboard-user.md (hunk 1 above)
Comment (shichun-0415) [reply dashboard/dashboard-user.md:57]:
```suggestion
- 当所连接的 TiDB 服务器启用了[安全增强模式 (SEM)](/system-variables.md#tidb_enable_enhanced_security) 时，先关闭 SEM，然后执行以下示例 SQL 语句创建一个允许登录 TiDB Dashboard 的 SQL 用户 `dashboardAdmin` ，创建完成后，再重新开启 SEM：
```

**Review by TomShawn — APPROVED** [review]

**Pull request merged**